tutorials/micro/micro_tflite.py

```diff
@@ -224,7 +224,7 @@
 #     from tvm.micro.contrib import zephyr
 #
 #     repo_root = subprocess.check_output(["git", "rev-parse", "--show-toplevel"], encoding='utf-8').strip()
-#     project_dir = f"{repo_root}/tests/micro/qemu/zephyr-runtime"
+#     project_dir = f"{repo_root}/apps/microtvm/zephyr/demo_runtime"
```

Comment:
nit: could you please use os.path.join(repo_root, "apps", "microtvm", ...) instead?

Reply:
@mehrdadh Sure, done for v2. I also included the new .conf file for disco board into tests/lint/check_file_type.py (I forgot it the first time and CI warned about it. BTW, I  pushed with `-f` v2 so we don't get the additional lines in the commit message when the commits are squashed. I think that's really confusing on TVM project and I'll bring it up on our next community meeting.